calwf3_recal_tvb.ipynb

```diff
@@ -0,0 +1,739 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    n, bins, patches = ax2.hist(b9q_data.flatten(), bins=200,range=(1,2))
</pre><p>Might it be more robust to give these variables different names than the ones from <code>ax1</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>